<launch>
  <!-- launches: startup sequence -->
  <!-- Written by Anass Al-Wohoush, July 16, McGill Robotics -->

  <include file="$(find planner)/launch/planner_node.launch" >
    <arg name="isInSim" value="0"/>
  </include>

  <arg name="timeout" default="35"/>
  <arg name="timer" default="5"/>
  <arg name="go" default="true"/>

  <node name="countdown" pkg="state_estimation" type="reinitialize_imu.py" respawn="false" output="screen">
    <param name="timeout" type="int" value="$(arg timeout)"/>
    <param name="timer" type="int" value="$(arg timer)"/>
    <param name="go" type="bool" value="$(arg go)"/>
  </node>
</launch>
